docs/adguard-browser-extension/solving-problems/debug-rules.md

```diff
@@ -0,0 +1,96 @@
+---
+title: How to debug rules
```

Comment:
How to debug rules where? There are several versions of the browser extension, only one of them is MV3 and this article is specifically about MV3 version.